docs/snowflake-native-apps/facebook-marketing.md

```diff
@@ -21,8 +21,11 @@ Do not refresh the Apps page while the application is being installed. This may
 2. On the left sidebar, click `Marketplace`.
 3. Search for `Facebook Marketing Connector` by Airbyte or navigate to https://app.snowflake.com/marketplace/listing/GZTYZ9BCRT8/airbyte-facebook-marketing-connector-by-airbyte
 4. Click `Request`. This will send a request that we will manually service as soon as we can.
+
+![](./facebook-marketing-shared-application.png)
+
 5. On the left sidebar, click `Apps`.
-6. Under the `Recently Shared with You` section, you should see the `Facebook Marketing Connector by Airbyte`. Click `Get`.
+6. Under the `Recently Shared with You` section, you should see the `Airbyte Facebook Marketing Connector` by Airbyte. Click `Get`.
```

Comment:
This process could change depending on what the process becomes when we start using the marketplace listing